### ISSUE TITLE
Add krdp to desktop image

### ISSUE THREAD

**Issue by looselyrigorous** [ublue-os/bazzite#1259]
### Describe the package

Hey,

I'd like to see the `krdp` package included. It's necessary for KDE Plasma 6.1's [Remote Desktop support](https://kde.org/announcements/plasma/6/6.1.0/#access-remote-plasma-desktops) to work. It seems like it's opt-in so it shouldn't affect existing users.

Thank you.

### Image

Desktop

**Comment by KyleGospo** [comment]
Added

**Comment by OGNeo** [comment]
Could this be added to the HTPC image too please?

**Comment by KyleGospo** [comment]
It was, but we pulled it due to it crashing. We will revisit this eventually. For now you can `rpm-ostree install krdp` and let us know how it works for you.

**Comment by OGNeo** [comment]
thank you.  installed but can't get it to connect.  Looks to be a widespread krdp issue though.

**Comment by tomtchea** [comment]
Was able to get KRDP working after allowing 'rdp' through the firewall. My setup is still fairly vanilla, most of my settings are default. So my NIC's firewall zone was set to 'FedoraWorkstation'. YMMV if you're messed with your Firewall or Network settings.

1. Open 'Firewall Configuration' in Applications and authenticate
2. Select your NIC on the left pane, and take note of the Zone name underneath the NIC name
3. On the right pane, select the Zone tab and select the Zone name that your NIC is using
4. On the right pane (next to the Zone window pane), select the Service tab
5. Under the Service tab, look under the list of services - specifically for 'rdp' and toggle the box to the 'on' position
6. Go back to your KRDP settings, in System Settings and restart the service (if it's was turned on already)

From your RDP client, try to reconnect to your KRDP server a couple times. It failed for me the first two times, but finally went through on the 3rd try. Like my RDP client would show the authentication/login box but the connection would eventually fail, after accepting the certificate.

The only thing I had modified during my KRDP setup was increasing the connection quality to favor max quality and used WiFi and Ethernet in separate testing sessions.

Service is fairly performant for me, when my handheld (ROG Ally X) is on Ethernet (RealTek RTS5411) - I'm using Windows 11's Remote Desktop Connection client.

One quirk I've noticed so far is that my RDP client couldn't connect to the KRDP server if both my WiFi and Ethernet were active. Once I disconnected the WiFi, I also needed to restart the KRDP service as well. I was able to connect to the KRDP service again.

I also verified that RDP over to the WiFi NIC worked fine as well (without the Ethernet NIC connected).
